src/app/src/App.js

```diff
@@ -54,6 +78,7 @@ function mapStateToProps(state) {
         selectedSensor: state.app.selectedSensor,
         sensors: state.map.sensors,
         isSensorModalDisplayed: state.app.isSensorModalDisplayed,
+        dispatch: state.app.dispatch,
```

Comment:
I didn't realize this would work. `dispatch` should just innately be available as a prop by virtue of the `connect` function call without having to get it in `mapStateToProps`.

Alternately, we've been using `mapDispatchToProps` to create redux action functions outside of the component without having to pass in dispatch.